images/vm-image-builder/start_libvirtd.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KubeVirt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+chown root:kvm /dev/kvm
+/usr/sbin/libvirtd -f /etc/libvirt/libvirtd.conf > /var/log/libvirtd.log 2>&1 &
+/usr/sbin/virtlogd -f /etc/libvirt/virtlogd.conf > /var/log/virtlogd.log 2>&1 &
```

Comment:
Using `-f /etc/libvirt/...` shouldn't be necessary, as those are the configuration files that get loaded by default.